cirq-superstaq/README.md

```diff
@@ -4,19 +4,7 @@ This package is used to access Superstaq via a Web API through [Cirq](https://gi
 Cirq programmers can take advantage of the applications, pulse level optimizations, and write-once-target-all
 features of Superstaq with this package.
 
-
-Please note that Python version `3.8` or higher is required. `cirq-superstaq` and all of its
-dependencies can be installed via:
-
-```
-python3 -m venv venv_cirq_superstaq
-source venv_cirq_superstaq/bin/activate
-pip install cirq-superstaq
-
-# Run the following to install dev requirements (required if you intend to run checks locally)
-pip install .[dev]
-
-```
+Please note that Python version `3.8` or higher is required. See installation instructions [here](https://github.com/Infleqtion/client-superstaq#readme).
```

Comment:
Just curious, the [link](https://github.com/Infleqtion/client-superstaq#readme) doesn't open the `cirq-superstaq` instructions tab directly and it also doesn't have the comment clarifying dev requirements install -- "# Run the following to install dev requirements (required if you intend to run checks locally)". Would it be beneficial to keep the instructions here as well for fewer redirects?

Reply:
I put the link in there so we have to update fewer files when we update instructions. Is it better to have fewer redirects or fewer files to update?

Reply:
That's a fair point if we expect to update it then I'd take fewer files to update side.

Reply:
is it worth keeping a one line user-installation (which isn't likely to change) example here, to emphasize that this is its own package (and also make it seem a little more accessible than having to click a link for installation instructions)? e.g. something like:

cirq-superstaq is [available on PyPI](https://pypi.org/project/cirq-superstaq) and can be installed with:
```
pip install cirq-superstaq
```
For further installation instructions, see [here](https://github.com/Infleqtion/client-superstaq#readme).

Reply:
Makes sense @richrines1! I just updated the files, so let me know if it looks alright

Reply:
nice, yep lgtm!